samples/snippets/publisher.py

```diff
@@ -273,6 +273,82 @@ def publish_messages_with_retry_settings(project_id, topic_id):
     # [END pubsub_publisher_retry_settings]
 
 
+def publish_with_ordering_keys(project_id, topic_id):
+    """Publishes messages with ordering keys."""
+    # [START pubsub_publish_with_ordering_keys]
+    from google.cloud import pubsub_v1
+
+    # TODO(developer): Choose an existing topic.
+    # project_id = "your-project-id"
+    # topic_id = "your-topic-id"
+
+    publisher_options = pubsub_v1.types.PublisherOptions(
+        enable_message_ordering=True
```

Comment:
Can we also add use of a regional endpoint, e.g., us-east1-pubsub.googleapis.com:443? Using a regional endpoint for ordering is a best practice.

Reply:
Done.